pennylane_lightning/core/src/simulators/lightning_qubit/StateVectorLQubit.hpp

```diff
@@ -69,10 +68,19 @@ class StateVectorLQubit : public StateVectorBase<PrecisionT, Derived> {
     using GeneratorKernelMap =
         std::unordered_map<GeneratorOperation, KernelType>;
     using MatrixKernelMap = std::unordered_map<MatrixOperation, KernelType>;
+    using ControlledMatrixKernelMap =
+        std::unordered_map<ControlledMatrixOperation, KernelType>;
+    using ControlledGeneratorKernelMap =
+        std::unordered_map<ControlledGeneratorOperation, KernelType>;
+    using ControlledGateKernelMap =
+        std::unordered_map<ControlledGateOperation, KernelType>;
```

Comment:
```suggestion
    using ControlledGateKernelMap =
        std::unordered_map<ControlledGateOperation, KernelType>;
    using ControlledGeneratorKernelMap =
        std::unordered_map<ControlledGeneratorOperation, KernelType>;
    using ControlledMatrixKernelMap =
        std::unordered_map<ControlledMatrixOperation, KernelType>;
```
I'm just suggesting keeping it in the same order as the no controlled to tidy it up.

Reply:
I tried to, but I guess I still missed quite a few spots. Thanks for pointing them out.